modules/integration/tests-integration/tests-transport/src/test/resources/testng.xml

```diff
@@ -44,35 +44,35 @@
     </test>
 
 
-    <test name="nhttp-Transport-Test" preserve-order="true" verbose="2">
-        <classes>
-            <!--The following test cases would change the server configuration when starting the test case,-->
-            <!--and restore the same once it's done-->
-            <!--If your test case needs to explicitly change the server configuration, add it here-->
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.DefaultRequestContentTypeTestCase"/>
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.ResponseAfterNttpEnabledTestCase"/>
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.NhttpMaximumOpenConnections"/>
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.ConfiguringNhttpAccessLogLocationTestCase"/>
-
-            <!--The SetNhttpAxis2Transport class would set the nhttp axis2 transport and restart the ESB-->
-            <!--The test cases that follow it would not change any server configurations-->
-            <!--RemoveNhttpAxis2Transport would restore the ESBs default transport-->
-            <!--If your test case doesn't need to change the server configuration, add it here-->
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.SetNhttpAxis2Transport"/>
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.ContentTypeCharsetTestCase"/>
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.MessageWithoutContentTypeTestCase"/>
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.OCSPCertificateValidationTestCase"/>
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.RetrieveBackendWsdlTestCase"/>
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.SetHostHttpHeaderTestCase"/>
-            <class name="org.wso2.carbon.esb.nhttp.transport.test.RemoveNhttpAxis2Transport"/>
-        </classes>
-    </test>
+    <!--<test name="nhttp-Transport-Test" preserve-order="true" verbose="2">-->
```

Comment:
Following is the created JIRA to fix the failing test cases.
https://wso2.org/jira/browse/ESBJAVA-3800